### DISCUSSION TITLE
PVP Screenshot

### DISCUSSION THREAD

**Discussion by hoojtiddy** [runelite/runelite#17326] [Ideas]
Just wondering the the PVP screen shot could be taken on the killing blow as opposed to once they are dead. The screenshot of a key going into your invent is not that interesting where as some last hits are.
![686786](https://github.com/runelite/runelite/assets/155387369/a8d48fe7-afb2-4ea1-8e4c-076371f24c0a)
![7686](https://github.com/runelite/runelite/assets/155387369/73c58ad5-3f8d-4df1-8e1a-13518e8e8a8c)

**Comment by iProdigy** [comment]
There's at least one hub plugin that does that (but submits the screenshot to a webhook like Discord rather than only saving locally)
